config/routes.rb

```diff
@@ -0,0 +1,3 @@
+UiComponents::Engine.routes.draw do
```

Comment:
That's true, but not my point. This gem is a rails engine, so everything in `config` and `app` will also be autoloaded into the target app. If you are not supposed to run the styleguide directly in the target app, it should not be there, right? `rake styleguide` still works then because it uses the dummy app.